www/docs/en/dev/config_ref/index.md

```diff
@@ -39,222 +39,357 @@ Note that before version 3.3.1-0.2.0, the file existed at `app/www/config.xml`,
 and that having it here is still supported.
 
 When using the CLI to build a project, versions of this file are
-passively copied into various `platforms/` subdirectories, for example:
+passively copied into various `platforms/` subdirectories.
+For example:
 
         app/platforms/ios/AppName/config.xml
         app/platforms/blackberry10/www/config.xml
         app/platforms/android/res/xml/config.xml
 
-This section details global and cross-platform configuration options.
-See the following sections for platform-specific options:
-
-- [iOS Configuration](../guide/platforms/ios/config.html)
-- [Android Configuration](../guide/platforms/android/config.html)
-- [BlackBerry 10 Configuration](../guide/platforms/blackberry10/config.html)
-
 In addition to the various configuration options detailed below, you
 can also configure an application's core set of images for each target
 platform. See [Icons and Splash Screens](images.html) for more information.
 
-## Core Configuration Elements
-
-This example shows the default `config.xml` generated by the CLI's
-`create` command, described in [The Command-Line Interface](../guide/cli/index.html):
-
-        <widget id="com.example.hello" version="0.0.1">
-            <name>HelloWorld</name>
-            <description>
-                A sample Apache Cordova application that responds to the deviceready event.
-            </description>
-            <author email="dev@callback.apache.org" href="http://cordova.io">
-                Apache Cordova Team
-            </author>
-            <content src="index.html" />
-            <access origin="*" />
-        </widget>
+# widget
+   Root element of the config.xml document.
+    
+   Attributes(type) | Description
+   ---------------- | ------------
+   id(string) | *Required* <br/> Specifies the app's reverse-domain identifier, and the `version` its full version number expressed in major/minor/patch notation.
+   version(string) | *Required* <br/> Full version number expressed in major/minor/patch notation.
+   versionCode(string) | **Platforms supported: Android** <br/> Alternative version for Android.
+   CFBundleVersion(string) |  **Platforms supported: iOS** <br/> Alternative version for iOS.  
```

Comment:
It will be great to Apple docs here: https://developer.apple.com/library/ios/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102364